per-seat-subscriptions/client/script.js

```diff
@@ -707,7 +707,7 @@ function showSubscriptionInformation() {
       accountInfo.subscriptionId = subscriptionId;
       accountInfo.priceId = currentPrice;
       accountInfo.quantity = currentQuantity;
-      accountInfo.customerId = latestInvoice.customer;
+      accountInfo.customerId = latestInvoice.customer.id;
```

Comment:
hmm. i think this might vary based on the server language :/ I think for dynamic languages, the customer's ID is in `latestInvoice.customer` and for the strongly typed servers like Go, it's in the `latestInvoice.customer.id`. Will think about how to solve, we might need a check to see if `customer` is a string

Reply:
I see. This seems to depend on the library implementation for each language.
https://stripe.com/docs/api/invoices/object?lang=php#invoice_object-customer

I understand that if we specify customer as expand in the API, it will be `customer.id`, otherwise customer will be `customerId`.

I'm sorry, but I just don't know how to fix it, so I'm going to close this PR.